app/models/manageiq/providers/container_manager.rb

```diff
@@ -30,5 +32,10 @@ def aggregate_cpu_total_cores(targets = nil)
     def aggregate_memory(targets = nil)
       aggregate_hardware(:computer_systems, :memory_mb, targets)
     end
+
+    def add_default_policies
+      # make sure that add_policy indeed adds a policy profile (aka MiqPolicySet)
+      add_policy(MiqPolicySet.find_by_guid('747a70d0-a8bf-4b63-bdad-1676bc0c5356'))
```

Comment:
Sounds good, thanks @moolitayer.  Note, we're all learning how this should work.  We want PRs reviewed and not blindly merged so that requires some knowledge of how to keep from making a PR too complex... while also not making it so hard to follow the full feature.  Clearly, though, having PRs depend on other PRs that are likely to change is very hard to maintain so it's always good to try to get the "skeleton" of a feature in first before trying to get other "more complicated" features in so you have less PR interdependencies.   I hope that makes sense.